Packs/PAN-OS/ReleaseNotes/2_1_12.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Palo Alto Networks PAN-OS
+
+- Added the argument **show-detail** to the commands **pan-os**, and **pan-os-query-logs**, which gives the ability to return the full data of the fields *after-change-preview*, and *before-change-preview*, under the fields *after-change-detail*, and *before-change-detail* in the raw response.
```

Comment:
I can see it was only added in 1 place.

```suggestion
- Added the argument **show-detail** to the commands **pan-os**, and **pan-os-query-logs**, to retrieve the full data of the fields *after-change-preview*, and *before-change-preview*, under the fields *after-change-detail*, and *before-change-detail* in the raw response.
```

Reply:
True, only to the command pan-os-query-logs. Fixed it